zboxcore/sdk/sdk.go

```diff
@@ -781,6 +781,46 @@ func GetAllocation(allocationID string) (*Allocation, error) {
 	return allocationObj, nil
 }
 
+func GetAllocationUpdates(allocation *Allocation) error {
```

Comment:
@lpoli May I know where is it used? And why it's returning an error instead of an Allocation object?

Reply:
It is actually needed when running tools like s3-migration, 0fs. These tools takes first instance of allocation object which initializes in itself upload, download, commit, etc channels.
Now while uploading files its better to know in advance for example; used size.

Allocation can also be updated by user with zbox cli(`zbox updateallocation`) and that information might be relevant for other applications as well.

If only single process would communicate with an allocation then it is not required but there can be multiple processes communicating thus changing allocation.

Returning allocation object is not necessary because this function receives pointer of original allocation instance. Error is to notify caller of the occurred error.